.github/workflows/nightly.yml

```diff
@@ -59,3 +59,35 @@ jobs:
         echo build --stamp >>.bazelrc.local
         ./go prep-release-zip
       nightly-release-files: build/dist/*.*
+
+  python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install Twine and Update .pypirc
+        run: |
+          pip install --upgrade pip
+          pip install --upgrade twine
+          echo "[distutils]" > ~/.pypirc
+          echo "index-servers =" >> ~/.pypirc
+          echo "    testpypi" >> ~/.pypirc
+          echo "[testpypi]" >> ~/.pypirc
+          echo "username:$PYPI_TOKEN" >> ~/.pypirc
+          echo "password:$PYPI_PASS" >> ~/.pypirc
+        env:
+          PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+          PYPI_PASS: ${{ secrets.PYPI_PASS }}
+      - name: Update Version
+        run: |
+          chmod +x scripts/github-actions/update-python-version.sh
+          ./scripts/github-actions/update-python-version.sh
+      - name: Build and Upload
+        run: |
+          bazel build --stamp //py:selenium-wheel //py:selenium-sdist
+          twine upload bazel-bin/py/selenium-$NIGHTLY_VERSION-py3-none-any.whl bazel-bin/py/selenium-$NIGHTLY_VERSION.tar.gz
```

Comment:
Don't we need to point to `https://test.pypi.org/project/selenium/` when we do this upload?

Reply:
Followed the steps outlined in the wiki https://github.com/SeleniumHQ/selenium/wiki/Releasing-Selenium#python which totally could be out of date